### PR TITLE
[new] Add bootstrap-confirmation 1.0.0, 1.0.1 and 1.0.2 with git auto-update, close #5069

### LINKED ISSUE #5069: [Request] Add bootstrap-confirmation
https://github.com/tavicu/bs-confirmation

### PR DESCRIPTION
**Pull request by MisatoTremor**
Current Project URL: https://github.com/tavicu/bs-confirmation

If checking for popularity, please also consider the original, now discontinued, project at https://github.com/ethaizone/Bootstrap-Confirmation

### CODE REVIEW

File: ajax/libs/bootstrap-confirmation/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      "type": "git",
+      "url": "https://github.com/tavicu/bs-confirmation.git"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tavicu/bs-confirmation.git",
+    "basePath": "/",
+    "files": [
+      "*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/bootstrap-confirmation/package.json]: It's not safe to use wildcard in non-dist folder.

**Comment by PeterDaveHello** [comment]
Would you also please squash them into only one commit? Thanks.

File: ajax/libs/bootstrap-confirmation/package.json (hunk 1 above)
Comment (MisatoTremor) [inline ajax/libs/bootstrap-confirmation/package.json]: @PeterDaveHello As the library is just one file, should I then just put that one under "files"?

Comment (PeterDaveHello) [inline ajax/libs/bootstrap-confirmation/package.json]: Yes.

**Comment by MisatoTremor** [comment]
@PeterDaveHello Sorry for having 2 commits earlier. I had the Sortable issue, but got it resolved now. Done

**Comment by PeterDaveHello** [comment]
Thanks.

**Pull request merged**